photon-ml/src/main/scala/com/linkedin/photon/ml/evaluation/EvaluatorType.scala

```diff
@@ -72,9 +67,12 @@ object EvaluatorType {
     case PoissonLoss.name | "POISSONLOSS" => PoissonLoss
     case SmoothedHingeLoss.name | "SMOOTHEDHINGELOSS" => SmoothedHingeLoss
     case SquaredLoss.name | "SQUAREDLOSS" => SquaredLoss
```

Comment:
Yes I agree, and I think the list will grow.

As a side note: here it looks redundant (e.g., ```PoissonLoss.name``` and ```"POISSONLOSS"```) partially because we want to keep it backward incompatible.